source/magpie/operation.d

```diff
@@ -203,6 +204,72 @@ auto merge(JoinTypes type = Inner, T, U)(T df1, U df2, string lsuffix = "x_", st
     else static assert(0, "Invalid join type. Available join type: left, right, outer, inner(default)");
 }
 
+enum AggregateOP
+{
+    count = 1,
+    max = 2,
+    min = 3,
+    mean = 4,
+    median = 5
+}
+
+auto aggregate(int axis, T, Ops...)(T df, Ops ops)
+    if(isDataFrame!T)
+{
+    static if(axis)
```

Comment:
if calling this with `axis==0` does nothing, it probably makes no sense and should be disallowed.

`auto aggregate(int axis,...)(...) if(isDataFrame!T && axis)

Reply:
scratch that axis is unused remove it.